bom/application/pom.xml

```diff
@@ -341,6 +341,15 @@
                 <type>pom</type>
             </dependency>
 
+            <!-- Smallrye Reactive Messaging dependencies, imported as a BOM -->
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
```

Comment:
Not a great idea. Complexify the productization.

Reply:
Ok I rolled back, we can hold the discussion later